ScratchDjango/User/forms.py

```diff
@@ -14,6 +15,22 @@ class Meta:
 class LoginForm(forms.ModelForm):
     password = forms.CharField(widget=forms.PasswordInput())
 
+
     class Meta:
         model = User
         fields = ["email", "password"]
+
+
+class ResetForm(forms.ModelForm):
+    password = forms.CharField(widget=forms.PasswordInput())
+    confirm_password = forms.CharField(widget=forms.PasswordInput())
+
+    class Meta:
+        model = User
+        fields = ("password", "confirm_password")
+
+
+class ResetEmailForm(forms.ModelForm):
+    class Meta:
+        model = User
+        fields = ["email"]
```

Comment:
I think it would be better to add clean_field method here.

Reply:
Ok